src/data.yaml

```
@@ -1,4 +1,23 @@
 jobs:
+  - name: Zerofy
+    website: https://www.zerofy.net
+    jobs: https://zerofy.recruitee.com
+    description: Carbon tracking and reduction for households / consumers.
+    remote: true
+    speculative: false
+    field: Carbon Analysis
```

```suggestion
    field: Carbon Analysis, Capture and Offsetting
```
This needs to be changed, otherwise a new category would be opened.